clang/tools/clang-scan-deps/Opts.td

```diff
@@ -34,6 +34,7 @@ def deprecated_driver_command : F<"deprecated-driver-command", "use a single dri
 defm resource_dir_recipe : Eq<"resource-dir-recipe", "How to produce missing '-resource-dir' argument">;
 
 def print_timing : F<"print-timing", "Print timing information">;
+def no_print_timing_header : F<"no-print-timing-header", "Do not print the timing information header">;
```

Comment:
Is this something you've found useful in practice? Naively I would look for a shell one-liner like piping to `sed 1d`

Reply:
You're right, this is easy to do outside of this tool. I have removed the option.